config/Config-kernel.in

```diff
@@ -20,6 +20,10 @@ config KERNEL_BUILD_DOMAIN
 	  returned by 'uname -a' on running systems.
 	  If not set, uses system hostname at build time.
 
+config KERNEL_TTY
```

Comment:
I think this is not needed, we set `CONFIG_TTY=y` already in the `target/linux/generic/config-5.15` configuration. It is always active. I also do not see any target which deactivated it.

Reply:
@hauke done